proxy/coco/src/state.rs

```diff
@@ -163,14 +163,15 @@ where
     let owner = default_owner(peer).await?.ok_or(Error::MissingOwner)?;
     Ok(peer
         .using_storage(move |store| {
-            replication::replicate(
-                store,
-                config,
-                Some(owner),
+            let fetcher = librad::git::storage::fetcher::PeerToPeer::new(
                 urn.clone(),
                 remote_peer,
                 addr_hints,
             )
+            .build(&store)
+            .unwrap()
```

Comment:
I made some changes in `daemon` that might be worth porting over https://github.com/radicle-dev/radicle-link/pull/576/commits/3484e241fadaa9e3109182c48a27a6fea8e014df

Reply:
052509d